config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
-  # get '/merchants/:merchant_id/dashboard', to: 'dashboard#index'
+
+  namespace :admin do 
+    resources :dashboard, only: [:index]
+    resources :merchants, only: [:index, :show] 
+    resources :invoices, only: [:index, :show]
+  end 
 
   resources :merchants do 
```

Comment:
These routes look good I think! I'm a little fuzzy on the nesting still, but looks like we are using full advanced routing which is great!

Reply:
I accidentally added the only to ones we don't need, so I got rid of that. Thank you!!!